src/cmd/src/metasrv.rs

```diff
@@ -100,6 +100,9 @@ struct StartCommand {
     http_timeout: Option<u64>,
     #[clap(long, default_value = "GREPTIMEDB_METASRV")]
     env_prefix: String,
+    /// The working home directory of this metasrv instance.
+    #[clap(short('d'), long)]
```

Comment:
`-d` often means running in the background. Maybe don't use any short names or change it to `-h` would be better.